### PR TITLE
feat : profile page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- User Profile 
- logout option

### CODE REVIEW

File: src/frontend/pages/Notes/Profile/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useAuth } from "../../../contexts/auth-context";
+
+export const Profile = () => {
+  const {
+    authState: { userData },
+    logout,
+  } = useAuth();
+  console.log(userData);
```
Comment (ravikumar1002) [inline src/frontend/pages/Notes/Profile/Profile.jsx]: remove this console

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

File: src/frontend/pages/Notes/NoteHome/NoteHome.jsx [hunk 2]
```diff
@@ -27,6 +28,7 @@ export const NoteHome = () => {
         {asideSelected === "archive" && <Archive />}
         {asideSelected === "label" && <Labels />}
         {asideSelected === "trash" && <Trash />}
+        {asideSelected === "profile" && <Profile />}
```
Comment (shrey27) [inline src/frontend/pages/Notes/NoteHome/NoteHome.jsx:31]: you can use switch case to handle rendering of different components or follow the routing method

**Review by shrey27 — APPROVED** [review]
UI is working as expected
PR can be merged

**Pull request merged**